brew/Brewfile.symlink

```diff
@@ -0,0 +1,36 @@
+tap "homebrew/bundle"
+tap "homebrew/services"
+brew "bat"
+brew "broot"
+brew "defaultbrowser"
+brew "deno"
+brew "fd"
+brew "figlet"
+brew "fish"
+brew "fzf"
+brew "gh"
+brew "git-machete"
+brew "glow"
+brew "ijq"
+brew "jq"
+brew "make"
+brew "mas"
+brew "mosh"
+brew "optipng"
+brew "python@3.10"
+brew "python@3.9"
+brew "readline"
+brew "rbenv"
+brew "ripgrep"
+brew "shellcheck"
+brew "starship"
+brew "tree"
+brew "webp"
+brew "wget"
+cask "1password-cli"
+cask "font-m-plus-1-code"
+cask "font-m+-nerd-font"
+cask "font-monoid-nerd-font"
+cask "font-symbols-only-nerd-font"
+cask "font-victor-mono-nerd-font"
```

Comment:
You may want to specify your choice of developer font here (`font-fira-code` and maybe `font-fira-code-nerd-font`).